### DISCUSSION TITLE
Fatal Error: Cannot read configuration file

### DISCUSSION THREAD

**Discussion by dextertd** [todotxt/todo.txt-cli#336] [Q&A] [answered]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
```
$ todo.sh add "test task"
Fatal Error: Cannot read configuration file /home/dexter/.todo/config
```

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

Fresh install of Manjaro KDE, install `todotxt` from AUR. Tested on two machines.

**What is the expected behavior?**

Creation of task.

**Which versions todo.sh are you using?**
> Run `todo.sh -V`

`TODO.TXT Command Line Interface v2.12.0`

**Which Operating System are you using?**

Linux Manjaro KDE Plasma (minimal)

**Which version of bash are you using?**
> Run `bash --version`

`GNU bash, version 5.1.0(1)-release (x86_64-pc-linux-gnu)`

**Comment by karbassi** [comment, marked as answer]
See troubleshooting page: https://github.com/todotxt/todo.txt-cli/wiki/Troubleshooting

Duplicate of #315
Duplicate of #252

**Comment by dextertd** [comment]
Thanks so much I'll check that out but I don't think this issue should be closed. The fact that this package throws an error on a clean Manjaro install should be looked into, although I apologise if this is a problem with the AUR package itself and better addressed to the maintainer.
